src/ray/protobuf/serve.proto

```diff
@@ -306,3 +306,21 @@ message ModelOutput {
 service RayServeBenchmarkService {
   rpc grpc_call(RawData) returns (ModelOutput);
 }
+
+// The parameters required to deploy a deployment, which are needed when deploying an
```

Comment:
```
message ApplicationDeploymentArgs {
  string application_name = 1
  DeploymentArgsList deployment_args = 2;
}
```
so that we can use them to trigger deploy/delete.

Reply:
cc: @edoakes for awareness.